sw/common/spi.c

```diff
@@ -0,0 +1,26 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include "spi.h"
+#include "dev_access.h"
+
+typedef struct spi_handle{
+    spi_reg_t *reg;
+    uint32_t speed;
+}spi_handle_t;
```

Comment:
Nit: just based on the vendored in code base, I think the consensus is adding a space between the type name and the curly braces.

Reply:
Fixed